pydax/_dataset.py

```diff
@@ -215,3 +226,37 @@ def load_dataset(name: str, *,
                                 f'\nCaused by:\n{e}')
 
     return dataset.data
+
+
+@handle_name_version_params
+def get_dataset_metadata(name: str, *,
+                         version: Optional[str] = None,
+                         human: bool = True) -> Optional[SchemaDict]:
+    """Print in human-readable form a dataset's metadata or return a copy of its schema.
+
+    :param name: Name of the dataset you want get the metadata of. You can get a list of these
+        datasets by calling :func:`pydax.list_all_datasets`.
+    :param version: Version of the dataset to load. Latest version is used by default. You can get a list of all
+        available versions for a dataset by calling :func:`pydax.list_all_datasets`.
+    :param human: Whether to print the metadata in human-readable form or to return a copy of the dataset's schema.
+        Defaults to True.
+    :return: A dictionary copy of the dataset's schema if ``human`` is set to False.
+    """
+
+    schema_manager = load_schemata()
+    dataset_schema = schema_manager.schemata['dataset_schema'] \
+                                   .export_schema('datasets', name, version)  # type: ignore [arg-type]
+    license_schema = schema_manager.schemata['license_schema'].export_schema('licenses')
+
+    if human:
+        print(f'Dataset name: {dataset_schema["name"]}\n'
+              f'Description: {dataset_schema["description"]}\n'
+              f'Size: {dataset_schema["estimated_size"]}\n'
+              f'Published date: {dataset_schema["published"]}\n'
+              f'License: {license_schema[dataset_schema["license"]]["name"]}\n'
+              f'Available subdatasets: {", ".join(dataset_schema["subdatasets"].keys())}'
+              )
```

Comment:
Can we return this string, which would be more natural? A program may redirect this string to wherever they see fit, such as some web page, etc. and they can print as they wish.